src/proj.rs

```diff
@@ -776,10 +776,9 @@ mod test {
     fn assert_almost_eq(a: f64, b: f64) {
         let f: f64 = a / b;
         assert!(f < 1.00001);
-        assert!(f > 0.99999);
+        assert!(f > 0.99999);    // This test should be disabled by default, as it requires network access
```

Comment:
I think moving this comment might have been unintentional?

Reply:
Hmm looks like rustfmt got confused. "We'll fix it in post"

Reply:
Fixed in 828d58e